adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala

```diff
@@ -141,20 +145,12 @@ sealed trait AlignmentRecordRDD extends AvroReadGroupGenomicRDD[AlignmentRecord,
   private[rdd] def maybeSaveBam(args: ADAMSaveAnyArgs,
                                 isSorted: Boolean = false): Boolean = {
 
-    if (args.outputPath.endsWith(".sam")) {
-      log.info("Saving data in SAM format")
-      saveAsSam(
-        args.outputPath,
-        asSingleFile = args.asSingleFile,
-        isSorted = isSorted
-      )
-      true
-    } else if (args.outputPath.endsWith(".bam")) {
-      log.info("Saving data in BAM format")
+    if (args.outputPath.endsWith(".sam") ||
+      args.outputPath.endsWith(".bam") ||
+      args.outputPath.endsWith(".cram")) {
```

Comment:
Would it be worth throwing all these file extension checks in the same utility class?  Or perhaps on a trait with a check method and a descriptive one (`getSupportedFileExtensions` or somthing similar)?